### PR TITLE
xbacklight: add page

### PR DESCRIPTION
**Pull request by dundalkian**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/xbacklight.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# xbacklight
+
+> Utility to adjust backlight brightness using RandR extension.
```
Comment (mebeim) [inline pages/linux/xbacklight.md]:
```suggestion
> Utility to adjust backlight brightness using the RandR extension.
```

I would add a "the" here.

**Comment by sbrl** [comment]
Nice one, @dundalkian! Now if only `xbacklight` worked on all my devices......

**Comment by dundalkian** [comment]
Agreed @sbrl, I run into that problem as well. As far as I understand it is only for intel devices, _with_ the intel video drivers, as it is the intel video drivers themselves that allow this to work.

It would be nice if this became a little more ubiquitous.

**Comment by sbrl** [comment]
Ah, I see. That does explain some things. Thanks @dundalkian :D

**Pull request merged**